examples/utils.py

```diff
@@ -226,6 +227,46 @@ def to_generation_fmt(x):
     return (train_stream, validation_stream, test_stream)
 
 
+def load_billsum_dataset() -> Tuple[caikit.core.data_model.DataStream]:
+    """Load the billsum dataset."""
+
+    def to_generation_fmt(x):
+        return GenerationTrainRecord(input=x["text"], output=str(x["summary"]))
+
+    dataset = datasets.load_dataset("billsum", split="ca_test")
+    train_test_dataset = dataset.train_test_split(test_size=0.2)
+    # # Split the 10% test + valid into half test, half valid
+    # test_valid = train_test_valid_dataset['test'].train_test_split(test=0.5)
+
+    build_stream = lambda split: caikit.core.data_model.DataStream.from_iterable(
+        [to_generation_fmt(datum) for datum in train_test_dataset[split]]
+    )
+    train_stream = build_stream("train")
+    validation_stream = build_stream("test")
```

Comment:
wanted to confirm if the "test" here is intentional vs. "validation"? same for the above financial phrasebook and not accidental copypaste

Reply:
Yep, since this dataset doesn't have "validation" set